src/App.tsx

```diff
@@ -14,6 +16,8 @@ const App: React.FC = () => {
         <Route path="/" element={<Home />} />{' '}
         <Route path="/register" element={<Register />} />{' '}
         <Route path="/login" element={<Login />} />{' '}
+        <Route path="/profile" element={<Profile />} />{' '}
+        <Route path="/profileoff" element={<ProfileOff />} />{' '}
```

Comment:
I think we should name this component UnauthenticatedProfile instead of ProfileOff.
When naming routes "kebab-case" must be used. Eg: unauthenticated-profile.

Reply:
I've made the changes you suggested! Thanks for pointing out the "UnauthenticatedProfile" and the route format. If there's anything else you want to add or suggest, I'm totally open to it! Thanks for the help!

Reply:
done